cpp/mysql_database_patcher.cc

```diff
@@ -92,7 +92,7 @@ void ApplyUpdate(DbConnection * const db_connection, const std::string &update_d
     SplitIntoDatabaseAndVersion(update_filename, &database, &update_version);
 
     if (not db_connection->mySQLDatabaseExists(database)) {
-        LOG_INFO("database \"" + database + "\" does not exist, skipping file " + update_filename);
+        LOG_WARNING("database \"" + database + "\" does not exist, skipping file " + update_filename);
```

Comment:
This shouldn't be a warning... there are many servers where this is totally normal, e.g. if no VuFind or IxTheo database exists on servers like nu, benu or ub28

Reply:
Reverted.